app.py

```diff
@@ -189,7 +238,7 @@ def signal_handler(sig, frame):
 
 def shutdown():
     logging.info("Cleaning up image files")
-    clean_up()
+    clean_up_user_images(None)
```

Comment:
why pass in `None` here?

Reply:
also, did you want to clear the cookie here?

Reply:
If `self` isn't set (ie `= None`) then it clears ALL temp file, not just the requesting user, which is what I want on shutdown. As for why `None` and not empty, python gives an error if I didn't pass `None`. And there's no cookie to clear here, it's a server side function, this is called when a user `Ctrl-C`s the app